app/email_templates/welcome.js

```diff
@@ -0,0 +1,30 @@
+const createWelcomeMail = customVariables => {
```

Comment:
as a pattern, let's make sure we destructure the vars object on the way in so we have an interface definition (or we can give it a type with ts)